### ISSUE TITLE
deDE Translation

### ISSUE THREAD

**Issue by neXter** [Zazou89/BetterBags_WorldEvents#2]
Thanks for the work! Would it be possibel to add German translations?

My proposal:

| en_US | de_DE | Comment |
|-------|-------|-------| 
| Remix | Remix | ofc |
| Meta Gems | Metaedelstein | based on retail ingame tooltip |
| Cogwheel Gems | Zahnradedelstein | based on retail ingame tooltip |
| Tinker Gems | Tüftleredelstein | based on retail ingame tooltip |
| Prismatic Gems | Prismatischer Edelstein | based on retail ingame tooltip |
| Consumables | Verbrauchbares | my proposal |
| Enhancements | Verbrauchbares: Verbesserungen | a bit tricky, as these are also consumables |
| Utilities | Verbrauchbares: Hilfsmittel | a bit tricky, as these are also consumables |

**Comment by Zazou89** [comment]
Hi, thanks for your suggestion, translation management can become a nightmare! But as a non-English-speaking player myself, I understand that it's always nice to have addons in your own language. 

I've added a fairly basic translation for the time being. I'll see about reorganizing all this over the next few months.

To avoid confusion, I've renamed the **Consumables** category by **Potions and Bandages**. I translated it into German, based on the wowhead translation, does that work for you?

Let me know if everything works for you!
